### PR TITLE
fix: resolve warnings

### PR DESCRIPTION
**Pull request by chilingling**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Release Notes

- **New Features**
  - Added a `locale` property to enhance localization in the Meta Code Editor.
  - Introduced a new emitted event `changeGroup` in the Block Group component for improved interactivity.
  - Added `TinyTooltip` component to enhance tooltip functionality in Design Settings.

- **Improvements**
  - Simplified conditional rendering in the Main component to improve UI clarity.
  - Enhanced state management in the Main component by injecting `isCollapsed` from the parent context.
  - Updated the `labelWidth` property in Tabs Group Configurator for greater flexibility.

- **Bug Fixes**
  - Adjusted placeholder attributes in Border Group to ensure static values are used.

- **Refactor**
  - Streamlined component structure in various files by removing unnecessary variables and props.
  - Modified template structure in the Meta List Items component for better organization.
  - Updated prop types in Numeric Select to restrict placeholder input to strings only.
  - Removed the `extendAttrs` variable in ToolbarBaseButton and ToolbarBaseIcon for cleaner code.

### CODE REVIEW

File: packages/settings/styles/src/components/border/BorderGroup.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <numeric-select
           :name="getProperty(BORDER_RADIUS_PROPERTY.BorderTopLeftRadius).name"
           :numericalText="borderRadius.BorderTopLeftRadius"
-          :placeholder="0"
+          placeholder="0"
```
Comment (gene9831) [inline packages/settings/styles/src/components/border/BorderGroup.vue:41]: `NumericSelet.vue` 组件里的 `props.placeholder` 类型也可以一并改下，改成 `String`，目前是 `[String, Number]`

**Review by hexqi — APPROVED** [review]

**Review by rhlin — APPROVED** [review]
LGTM

File: packages/settings/styles/src/components/inputs/NumericSelect.vue [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default {
       default: false
     },
     placeholder: {
-      type: [String, Number],
+      type: [String],
```
Comment (hexqi) [inline packages/settings/styles/src/components/inputs/NumericSelect.vue]: 单个值没必要用数组

File: packages/common/component/toolbar-built-in/ToolbarBaseIcon.vue [hunk 3]
```diff
@@ -37,12 +36,6 @@ export default {
       type: Object,
       default: () => {}
```
Comment (hexqi) [inline packages/common/component/toolbar-built-in/ToolbarBaseIcon.vue]: 这里箭头函数返回值少了括号，`() => ({})`

**Pull request merged**